.gitignore

```diff
@@ -11,3 +11,4 @@ driver-test-data.tar.gz
 perf
 **mongocryptd.pid
 *.test
+**.md
```

Comment:
I usually keep a `design.md` file around and it's annoying to have to manually ignore it.